public/stylesheets/faq.css

```diff
@@ -25,8 +25,9 @@ figure figcaption {
 }
 
 .faq .faq-item h2 {
-    margin-top: 0;
+    margin-top: -70px;
```

Comment:
Could add the link to the place where you found the recommended fix as part of a comment? Also, could you answer this question:

Did you decide the constant based on height of the navbar or something else?